examples/3d/parenting.rs

```diff
@@ -1,7 +1,8 @@
 use bevy::prelude::*;
 
 /// This example illustrates how to create parent->child relationships between entities how parent
-/// transforms are propagated to their descendants
+/// transforms are propagated to their descendants.
+/// Any entity with a `Transform` _and_ a `GlobalTransform` can be a parent.
```

Comment:
I'm a little nervous about this line. Right now it's effectively true, but the longer-term vision is to enable the use of the parent-child hierarchy for other types of behavior.

Reply:
```suggestion
///  `Transform` can be inherited from any entity with both the `Transform` and `GlobalTransform` components.
```